migration/migration_test.go

```diff
@@ -141,11 +141,11 @@ func TestFailingMigration(t *testing.T) {
 	require.Equal(t, 0, v)
 }
 
-func TestRealDataBadger(t *testing.T) {
+func TestRealDataBadgerV0(t *testing.T) {
 	logger.SetDebugLogging()
 	_ = logger.SetLogLevel("badger", "error")
 	tmpDir := t.TempDir()
-	err := copyDir("testdata/badgerdump", tmpDir+"/badgerdump")
+	err := copyDir("testdata/badgerdumpv0", tmpDir+"/badgerdump")
```

Comment:
The `badgerdump` folder had some exported database. Now I needed a new one that was already migrated to version 1. So I renamed the old one with version number.